python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -493,6 +493,18 @@ py::object pyAltLaunchKernelR(const std::string &name, MlirModule module,
   return returnValue;
 }
 
+/// @brief Helper function to get boolean environment variable
+static bool getEnvBool(const char *envName, bool defaultVal = false) {
+  if (auto envVal = std::getenv(envName)) {
+    std::string tmp(envVal);
+    std::transform(tmp.begin(), tmp.end(), tmp.begin(),
+                   [](unsigned char c) { return std::tolower(c); });
+    if (tmp == "1" || tmp == "on" || tmp == "true" || tmp == "yes")
```

Comment:
Should this allow anything that starts with "t" or "y"? Or is that too permissive? CMake accepts "y", for example.

Reply:
added